src/ar/calibration.cpp

```diff
@@ -194,16 +199,32 @@ static bool runCalibration( vector<vector<Point2f> > imagePoints,
 
     vector<vector<Point3f> > objectPoints(1);
     calcChessboardCorners(boardSize, squareSize, objectPoints[0], patternType);
+    objectPoints[0][boardSize.width - 1].x = objectPoints[0][0].x + grid_width;
+    newObjPoints = objectPoints[0];
 
     objectPoints.resize(imagePoints.size(),objectPoints[0]);
 
-    double rms = calibrateCamera(objectPoints, imagePoints, imageSize, cameraMatrix,
-                    distCoeffs, rvecs, tvecs, flags|CALIB_FIX_K4|CALIB_FIX_K5);
-                    ///*|CALIB_FIX_K3*/|CALIB_FIX_K4|CALIB_FIX_K5);
+    double rms;
+    int iFixedPoint = -1;
+    if (release_object)
+        iFixedPoint = boardSize.width - 1;
+    rms = calibrateCameraRO(objectPoints, imagePoints, imageSize, iFixedPoint,
```

Comment:
Ok so I guess this is the new OpenCV4 API call? (Not actually a question, just sort of trying to prove I'm paying attention to what's going on.)

Small suggestion: instead of `using namespace cv` above, just use `cv::calibrateCameraRO` here. That makes it clear that this is an OpenCV method. We don't have to do that in this PR, but just as a suggestion for the future. "using namespace" usually makes code a bit less clear to unfamiliar readers.

Reply:
Yeah, that makes sense. OpenCV seems pretty eager to use `using namespace` in their sample code which isn't great practice, I agree. I can just modify that now.